principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**I. Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**II. Censorship resistance**
+We enable free flow of information. No content is under surveillance. We abide
+by the cryptoeconomic design principle of censorship resistance.
+
+**III. Security**
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**IV. Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as being a Pseudo-Anonymous platform. Additionally, we strive to provide the right of total anonymity.
```

Comment:
Should this include a section about privacy "by default"? Was it decided if this was a core principle? (I think it should be).

Reply:
@noman-land Do you think the current text doesn't reflect that? How would you suggest re-writing this text to reflect it?

Reply:
I think the current text reflects the "right" to privacy but not the goal of making it a default. Like how WhatsApp includes e2e encryption but it's off by default and you have to opt into it.

What about something like: 

_Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as being a pseudo-anonymous platform **that has maximum security enabled by default**. Additionally, we strive to provide the right of total anonymity._